library/src/main/java/com/chuckerteam/chucker/api/Chucker.kt

```diff
@@ -30,6 +37,34 @@ public object Chucker {
             .setFlags(Intent.FLAG_ACTIVITY_NEW_TASK)
     }
 
+    /**
+     * Create a shortcut to launch Chucker UI.
+     * @param context An Android [Context].
+     */
+    internal fun createShortcut(context: Context) {
+        if (Build.VERSION.SDK_INT >= Build.VERSION_CODES.N_MR1) {
+            run outside@{
+                context.getSystemService(ShortcutManager::class.java)?.let { sm ->
+                    sm.dynamicShortcuts.forEach {
+                        if (it.id == SHORTCUT_ID) return@outside
+                    }
+                    val shortcut = ShortcutInfo.Builder(context, SHORTCUT_ID)
+                        .setShortLabel(context.getString(R.string.chucker_shortcut_label))
+                        .setLongLabel(context.getString(R.string.chucker_shortcut_label))
+                        .setIcon(
+                            Icon.createWithResource(context, R.mipmap.chucker_ic_launcher_round)
+                        )
+                        .setIntent(getLaunchIntent(context).setAction(Intent.ACTION_VIEW))
+                        .build()
+                    try {
+                        sm.addDynamicShortcuts(listOf(shortcut))
+                    } catch (ignored: Throwable) {
+                    }
```

Comment:
NIT: Maybe log it? We have a logger in this project for such cases.

Reply:
Logged it.